infinispan-spring-boot-starter-remote/pom.xml

```diff
@@ -73,6 +79,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.10.0</version>
```

Comment:
can you use `${version.jackson.databind}`? 
So I don't have to worry about it downstream :)

Reply:
yes